pallet-logion-loc/src/tests.rs

```diff
@@ -512,7 +513,7 @@ fn it_fails_adding_file_when_insufficient_funds() {
         let file = FileParams {
             hash: sha256(&"test".as_bytes().to_vec()),
             nature: sha256(&"test-file-nature".as_bytes().to_vec()),
-            submitter: SupportedAccountId::Polkadot(LOC_REQUESTER_ID),
+            submitter: SupportedAccountId::Polkadot(LOC_OWNER1),
```

Comment:
Why change this?

Reply:
The signer of the extrinsic is now more strictly validated against given attribute `submitter`, hence those changes. In this particular case - testing funds - it' indeed better to sign and have submitter equal to requester (rather than owner).